test/drivers/test_cholqrcp.cc

```diff
@@ -41,6 +41,7 @@ class TestCholQRCP : public ::testing::Test
         RandLAPACK::CholQRCP<T> CholQRCP(false, false, seed, tol);
         CholQRCP.nnz = nnz;
         CholQRCP.num_threads = 32;
```

Comment:
The machines that we run the RandLAPACK tests on won't have 32 cores, in general. Set this to something more reasonable, like 4.

Reply:
nice catch. something like `CholQRCP.num_threads = std::max(std::thread::hardware_concurency()/4,2);` will make it machine independent.

Reply:
Fixed.